pages/integrations/github.md

```diff
@@ -119,8 +118,8 @@ You can customize the commit statuses, for example to reuse the same pipeline fo
         - github_commit_status:
             context: "my-custom-status"
     ```
-2. In _Pipeline Settings_ > _GitHub_, make sure _Update commit statuses_ is not selected. Note that this prevents Buildkite from automatically creating and sending statuses for this pipeline, meaning you will have to handle all commit statuses through the `pipeline.yml`.
-3. When you make a new commit or pull request, you should see _my-custom-status_ as the commit status:
+2. In **Pipeline** > your specific pipeline > **Settings** > **GitHub**, make sure **Update commit statuses** is not selected. Note that this prevents Buildkite from automatically creating and sending statuses for this pipeline, meaning you will have to handle all commit statuses through the `pipeline.yml`.
```

Comment:
Should these list items (and the ones in the list just below) both start with `1.` as well?

Reply:
Good pickup! Thanks!